jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -1036,7 +1036,7 @@ static void find_and_replace_parameter(
 /// `java::Outer::T` instead of `java::Outer$Inner::T`.
 /// \param type
 /// \param replacement_parameters
-static void find_and_replace_parameters(
+static: void find_and_replace_parameters(
```

Comment:
It seems the scripting got something wrong here, which is caused by the missing documentation of the parameters.